document/src/vespa/document/base/globalid.cpp

```diff
@@ -29,7 +32,7 @@ getHexVal(char c)
         return (c - 'A' + 10);
     }
     assert(validateHex(c));
-    abort();
```

Comment:
Here I guess there should in else instead with just a simple assert.

Reply:
the abort is needed to avoid "control reaches end of non-void function"